p2panda-discovery/README.md

```diff
@@ -1 +1,38 @@
-Awesome documentation goes here.
+<h1 align="center">p2panda-discovery</h1>
+
+<div align="center">
+  <img src="https://raw.githubusercontent.com/p2panda/.github/main/assets/panda-left.gif" width="auto" height="30px">
+  <strong>Peer discovery traits and services</strong>
+  <img src="https://raw.githubusercontent.com/p2panda/.github/main/assets/panda-right.gif" width="auto" height="30px">
+</div>
+
+<div align="center">
+  <h3>
+    <a href="https://p2panda.org">
+      Website
+    </a>
+  </h3>
+</div>
+
+This crate is used to share address information about peers on a network. It provides a generic
+interface for the definition of a discovery service, as well as an interface for organising a
+collection of such services. An mDNS discovery implementation is provided for peer discovery over
+local networks.
+
+## Features
```

Comment:
Not too sure about this. It's more of a "Main types" section and not a "Features" section. Maybe remove it entirely?

Reply:
I don't really see the benefit of having a "main types" section.

I think we can tell much more about the crate by listing the features in non-code terms, for understanding the types we have Rust docs.

Reply:
Agreed. I'll remove them.